### ISSUE TITLE
Orientation issue on both platforms

### ISSUE THREAD

**Issue by davee44** [myflashlab/fileChooser-ANE#23]
If the app is in the landscape mode, the file browser comes up in portrait, no matter what.

## Environment ##
* Android API Version: 24 (7.0)
* iOS Version: 11.3
* AIR SDK Version: 30

## Steps to Reproduce ##
1. Set the `aspectRatio` parameter in the XML descriptor file to either `portrait` or `any`:
2. Set the orientation manually in the AIR app:
`stage.setAspectRatio( StageAspectRatio.LANDSCAPE );`
3. Invoke the file chooser:
`fileChooser.choose( "*/*", "File uploading" );`

**Result**: the screen rotates sideways to the portrait mode.

**Expected result**: the screen remains in landscape.

**Comment by davee44** [comment]
@myflashlab yes, I'm using the version 4.1.4 and observe this issue on both Android and iOS platforms.

**Comment by myflashlab** [comment]
@davee44 can't duplicate this. everything works fine as it seems. help us understand the problem, maybe send a demo .apk to run here?

**Comment by davee44** [comment]
Ok, on Android it works correctly only if the system screen orientation is not locked to portrait. But if I lock the orientation and then the app forces it to be landscape (as described above), the ANE's browser still goes portrait even if the app is rotated sideways.

On iOS 11.3 it messes up the orientation even worse. The browser comes up in portrait (while the app is in landscape) and if I press Cancel in the browser – it closes and rotates the app upside down portrait (while it still meant to be landscape, as it was before opening the browser).
